nuget_witsml.md

```diff
@@ -0,0 +1,36 @@
+## Access pre-built Witsml nuget package
+
+If you only need to include the [Witsml](https://github.com/equinor/witsml-explorer/tree/main/Src/Witsml) project
+```csharp
+witsml-explorer/Src/Witsml
+```
+
+Witsml nuget package can be added to your own project using [nuget](https://docs.microsoft.com/en-us/nuget/). 
+It is necessary to author a new personal access token to be able to add the package to your project. A `nuget.config` (config file at solution, user or global scope) has to accommodate for username and the token afterwards.
+
+**Example** 
+
+1. Create your api access token at [https://github.com/settings/tokens](https://github.com/settings/tokens)
+2. Make those available through a config file at your preferred location (see: [configuring-nuget-behavior](https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior))
+3. Add Witsml package to your project
+   ```csharp
+   dotnet add PROJECT package Witsml --version 1.0.18
+   ```
```

Comment:
I would suggest to change the `command` to be of type "shell" rather than "csharp".
```suggestion
   ```sh
   dotnet add PROJECT package Witsml --version 1.0.18
   ```
```